etc/profile-m-z/mirage.profile

```diff
@@ -20,7 +20,6 @@ include disable-exec.inc
 include disable-interpreters.inc
 include disable-passwdmgr.inc
 include disable-programs.inc
-include disable-shell.inc
```

Comment:
We have [allow-bin-sh.inc](https://github.com/netblue30/firejail/commit/f18124baa4290e658babc6dd0716232e9a474335) now to handle this.

Reply:
Ah thanks, I just saw it.